src/Splat/Platforms/Android/Bitmaps/PlatformBitmapLoader.cs

```diff
@@ -28,11 +28,15 @@ static PlatformBitmapLoader()
         {
             // NB: This is some hacky shit, but on MonoAndroid at the moment,
             // this is always the entry assembly.
-            var assm = AppDomain.CurrentDomain.GetAssemblies()[1];
-
-            var resources = assm.GetModules().SelectMany(x => x.GetTypes()).First(x => x.Name == "Resource");
-
-            _drawableList = resources.GetNestedType("Drawable").GetFields()
+            var assembly = AppDomain.CurrentDomain.GetAssemblies()[1];
```

Comment:
Can we change the array syntax?

Reply:
What would you like to change it to?

Reply:
Could do `AppDomain.CurrentDomain.GetAssemblies().Skip(1).FirstOrDefault()` -- like the comment says pretty hacky regardless.

Reply:
Oh and also if you don't mind, lets not have swearing in our comments if we can help it I guess. Just change it to hacky ;0

Reply:
Yeah.  It just felt odd looking at the code seeing the array syntax and then right below it the linq syntax.  Might as well clean it up while we are in here.